server/index.js

```diff
@@ -19,10 +19,36 @@ const db = new pg.Pool({
 
 app.use(express.json());
 
+// ---------------------------- GET REQUESTS ---------------------//
+app.get('/api/accounts/new-ratings/:username', (req, res, next) => {
+  const sql = `
+    select round(avg("rating"), 2) as "updatedRating"
+    from "accounts"
+    left join "ratings"
+    ON "accountId" = "ratedWho"
+    Where "username" = $1
+    group by "accountId"
+  `;
+  const username = req.params.username;
+  const params = [username];
+  db.query(sql, params)
+    .then(result => {
+      if (!result.rows[0]) {
+        res.status(404).json({
+          error: 'No users found'
+        });
+      } else {
+        res.json(result.rows[0]);
+      }
+    })
+    .catch(err => next(err));
+});
```

Comment:
I don't understand what this is for. Does the server not already tell us the ratings of each user when we fetch the users? If not, can we adjust the server code to do that for us?